PancakeSwap-Bots/Set-Masterchef-Settings/src/network.ts

```diff
@@ -4,9 +4,6 @@ const CONFIG: AgentConfig = {
   56: {
     masterChef: "0x73feaa1eE314F8c655E354234017bE2193C9E24E",
   },
-  97: {
-    masterChef: "0xbD315DA028B586f7cD93903498e671fA3efeF506",
-  },
```

Comment:
you shouldn't remove this. otherwise the testnet txns in readme wouldn't work. it's better to keep this testnet network commented out and outside of this config object. you can refer to here https://github.com/NethermindEth/Forta-Agents/blob/main/Balancer-Bots/Large-Pool-Balance-Change/src/agent.config.ts